default.py

```diff
@@ -159,27 +154,17 @@ def display_weeks_games(self):
             if game['phase'] == 'FINAL' or game['phase'] == 'FINAL_OVERTIME':
                 # show game duration only if user wants to see it
                 if addon.getSetting('hide_game_length') == 'false':
-                    game_info = '%s [CR] Duration: %s' % (game['phase'], str(datetime.timedelta(seconds=int(float(game['video']['videoDuration'])))))
+                    game_info = '%s [CR] Duration: %s' % (game['phase'], str(timedelta(seconds=int(float(game['video']['videoDuration'])))))
                 else:
                     game_info = game['phase']
             else:
-                try:
-                    if addon.getSetting('local_tz') == '0':  # don't localize
-                        game_datetime = datetime.datetime(*(time.strptime(game['gameDateTimeUtc'], date_time_format)[0:6]))
-                        game_info = game_datetime.strftime('%A, %b %d - %I:%M %p')
-                    else:
-                        from_zone = tz.tzutc()
-                        to_zone = tz.tzlocal()
-                        game_datetime = datetime.datetime(*(time.strptime(game['gameDateTimeUtc'], date_time_format)[0:6]))
-                        game_datetime = game_datetime.replace(tzinfo=from_zone)
-                        local_time = game_datetime.astimezone(to_zone)
-                        if addon.getSetting('local_tz') == '1':  # localize and use 12-hour clock
-                            game_info = local_time.strftime('%A, %b %d - %I:%M %p')
-                        else:  # localize and use 24-hour clock
-                            game_info = local_time.strftime('%A, %b %d - %H:%M')
-                except:  # all else fails, just use their raw date value
-                    game_datetime = game['gameDateTimeUtc'].split('T')
-                    game_info = game_datetime[0] + '[CR]' + game_datetime[1].split('.')[0] + ' ET'
+                if addon.getSetting('time_notation') == '0':  # 12-hour clock
+                    datetime_format = '%A, %b %d - %I:%M %p'
+                else:  # 24-hour clock
+                    datetime_format = '%A, %b %d - %H:%M'
+
+                datetime_obj = gp.parse_datetime(game['gameDateTimeUtc'], True)
```

Comment:
It's in UTC now so that's why I thought it didn't make any sense to not localize it.